backend/lib/logger/Logger.js

```diff
@@ -49,11 +49,11 @@ class Logger {
     }
   }
 
-  request ({ id, uri, method, httpVersion = '1.1', user, headers, body }) {
+  request ({ id, url, method, httpVersion = '1.1', user, headers, body }) {
     if (!this.isDisabled(LEVELS.trace + 1)) {
```

Comment:
debug level and for response and log timestamp

Reply:
✔️